### DISCUSSION TITLE
Stream doesn't detect native queries?

### DISCUSSION THREAD

**Discussion by jihanga** [simolus3/drift#1571] [Q&A]
I use Room in android and SQLite in IOS.
And I connected native and  drift.

So, I expected the stream(watch func) to detect that native's query.
But stream is can't detect that.

When I manually call get func, data was updated.

I need native query.
So I think pooling. 
Is there any better way than this?

**Comment by westito** [comment]
In Drift watch function is internal (library level)

**Reply by jihanga** [reply]
I know.
So, I find other way.

**Reply by westito** [reply]
You can make a native bridge between Android and Flutter to send custom notifications (observe LiveData <-> query.watch())
